src/pages/settings/Subscription/CardSection/BillingBanner.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {View} from 'react-native';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import Text from '@components/Text';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type BillingBannerProps = {
+    title: string;
+    subtitle: string;
```

Comment:
Does it make sense to make this optional? Are we guaranteed to have subtitle all the time?

Reply:
updated